aim/web/ui/src/components/kit_v2/Slider/Slider.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import _ from 'lodash-es';
+
+import Text from '../Text';
+
+import { ISliderProps } from './Slider.d';
+import {
+  SliderLabel,
+  SliderMark,
+  SliderRange,
+  SliderRoot,
+  SliderThumb,
+  SliderTrack,
+} from './Slider.style';
+
+//TODO: Fix the thumb label overlapping issue
+
+const Slider = React.memo(
+  ({
+    min = 0,
+    max = 100,
+    value,
+    defaultValue = [10],
+    marks = [],
+    onValueChange,
+    ...props
+  }: ISliderProps) => {
+    const [sliderValue, setSliderValue] =
+      React.useState<number[]>(defaultValue);
+
+    React.useEffect(() => {
+      if (value) {
+        if (value && !_.isEqual(value, sliderValue)) {
+          setSliderValue(value);
```

Comment:
please remove first `if` condition, since `value` double checked

Reply:
Let's consider that `value` property could be `0` as well , and it's falsy value. 
And the condition will not pass if the value is equal to 0.